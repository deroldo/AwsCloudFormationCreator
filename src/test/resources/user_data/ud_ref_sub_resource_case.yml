MyData:
  Template: ref_sub_resource_case
  Foo: bar