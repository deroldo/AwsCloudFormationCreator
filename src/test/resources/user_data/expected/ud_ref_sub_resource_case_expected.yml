---
Resources:
  MyData-AnyResource:
    Type: "AnyAwsType"
    Properties:
      Foo: "foo=bar, other=${MyData-Other}"
  MyData-Other:
    Type: "AnyAwsType"
    Properties:
      Foo: "baaaar"

